subsurface-helper.cpp

```diff
@@ -76,8 +76,8 @@ void run_ui()
 	// same directory the executable was started from <bundle>/Contents/MacOS/
 	// To work around this we need to manually copy the components at install time
 	// to Contents/Frameworks/qml and make sure that we add the correct import path
-	QStringList importPathList = engine.importPathList();
-	Q_FOREACH (QString importPath, importPathList) {
+	const QStringList importPathList = engine.importPathList();
+	for (const QString &importPath: importPathList) {
```

Comment:
clang seems to fail here - and it seems to be right.
replace does change the original string.

Reply:
Ah, that as the reason ;)

Reply:
Fixed.